### PR TITLE
[pigeon] Run swift-format on ungenerated example app Swift files

### PR DESCRIPTION
**Pull request by jmagman**
Run `swift-format` linter/formatter on Swift files in example apps.  This only touches Swift files in example/test code.  

This also doesn't format any autogenerated Swift files, which would require changed in the pigeon generator: https://github.com/flutter/flutter/issues/141799

Part of turning on swift-format CI formatting https://github.com/flutter/packages/pull/5928

## Pre-launch Checklist

- [x] I read the [Contributor Guide] and followed the process outlined there for submitting PRs.
- [x] I read the [Tree Hygiene] wiki page, which explains my responsibilities.
- [x] I read and followed the [relevant style guides] and ran the auto-formatter. (Unlike the flutter/flutter repo, the flutter/packages repo does use `dart format`.)
- [x] I signed the [CLA].
- [x] The title of the PR starts with the name of the package surrounded by square brackets, e.g. `[shared_preferences]`
- [x] I listed at least one issue that this PR fixes in the description above.
- [x] I updated `pubspec.yaml` with an appropriate new version according to the [pub versioning philosophy], or this PR is [exempt from version changes].
- [x] I updated `CHANGELOG.md` to add a description of the change, [following repository CHANGELOG style].
- [x] I updated/added relevant documentation (doc comments with `///`).
- [x] I added new tests to check the change I am making, or this PR is [test-exempt].
- [x] All existing and new tests are passing.

If you need help, consider asking for advice on the #hackers-new channel on [Discord].

[Contributor Guide]: https://github.com/flutter/packages/blob/main/CONTRIBUTING.md
[Tree Hygiene]: https://github.com/flutter/flutter/wiki/Tree-hygiene
[relevant style guides]: https://github.com/flutter/packages/blob/main/CONTRIBUTING.md#style
[CLA]: https://cla.developers.google.com/
[flutter/tests]: https://github.com/flutter/tests
[breaking change policy]: https://github.com/flutter/flutter/wiki/Tree-hygiene#handling-breaking-changes
[Discord]: https://github.com/flutter/flutter/wiki/Chat
[pub versioning philosophy]: https://dart.dev/tools/pub/versioning
[exempt from version changes]: https://github.com/flutter/flutter/wiki/Contributing-to-Plugins-and-Packages#version-and-changelog-updates
[following repository CHANGELOG style]: https://github.com/flutter/flutter/wiki/Contributing-to-Plugins-and-Packages#changelog-style
[test-exempt]: https://github.com/flutter/flutter/wiki/Tree-hygiene#tests

### CODE REVIEW

**Review by hellohuanlin — APPROVED** [review]

**Comment by jmagman** [comment]
No CHANGELOG or version bump needed based on https://github.com/flutter/packages/pull/5933#discussion_r1458260704

**Review by tarrinneal — APPROVED** [review]
Thank you!

**Review by hellohuanlin — APPROVED** [review]

File: packages/pigeon/example/app/ios/Runner/AppDelegate.swift [hunk 1]
```diff
@@ -15,14 +15,14 @@ private class PigeonApiImplementation: ExampleHostApi {
   }
 
   func add(_ a: Int64, to b: Int64) throws -> Int64 {
-    if (a < 0 || b < 0) {
-      throw FlutterError(code: "code", message: "message", details: "details");
+    if a < 0 || b < 0 {
+      throw FlutterError(code: "code", message: "message", details: "details")
     }
```
Comment (tarrinneal) [inline packages/pigeon/example/app/ios/Runner/AppDelegate.swift:20]: This is the changed line of code from the README

File: packages/pigeon/example/app/ios/Runner/AppDelegate.swift [hunk 2]
```diff
@@ -15,14 +15,14 @@ private class PigeonApiImplementation: ExampleHostApi {
...
     return a + b
   }
 
   func sendMessage(message: MessageData, completion: @escaping (Result<Bool, Error>) -> Void) {
-    if (message.code == Code.one) {
+    if message.code == Code.one {
```
Comment (tarrinneal) [inline packages/pigeon/example/app/ios/Runner/AppDelegate.swift:25]: This too actually

File: packages/pigeon/example/app/ios/Runner/AppDelegate.swift (hunk 1 above)
Comment (jmagman) [reply packages/pigeon/example/app/ios/Runner/AppDelegate.swift:20]: Ah, thank you!

File: packages/pigeon/example/app/ios/Runner/AppDelegate.swift (hunk 2 above)
Comment (jmagman) [reply packages/pigeon/example/app/ios/Runner/AppDelegate.swift:25]: 👍

**Comment by tarrinneal** [comment]
I guess the version bump is needed to publish these changes to the README, but I don't think it's necessary as another pr will publish it very soon anyway. I'll let you decide. :)

**Comment by jmagman** [comment]
> I guess the version bump is needed to publish these changes to the README, but I don't think it's necessary as another pr will publish it very soon anyway. I'll let you decide. :)

I'm okay with the snippet whitespace changes going in the next version bump.

**Pull request merged**